### ISSUE TITLE
Tests for $.Class()

### ISSUE THREAD

**Issue by LeaVerou** [LeaVerou/bliss#179]
Just realized we have no tests for $.Class(). Any takers?

(whoever's interested, also see #177 )

**Comment by dperrymorrow** [comment]
Whoops! ill see if I can take care of that

**Comment by LeaVerou** [comment]
Thanks!!

**Comment by StrongRef** [comment]
@dperrymorrow Any progress?
I can take a look at it otherwise.

**Comment by dperrymorrow** [comment]
yeah i have not gotten to this @StrongRef apologies, its all yours if you have the time.
